Assets/Scripts/UI/Camera/SpectatorMovement.cs

```diff
@@ -50,6 +64,10 @@ private void Update()
             {
                 transform.position -= (Vector3) ((base.transform.forward * speed) * Time.deltaTime);
             }
+            if (InputManager.KeyDown(InputUi.Restart))
```

Comment:
Seems like here you are using InputUi.Restart. Do note, that the "this" modifier is not required and should rather be avoided in small classes.